airflow/sentry.py

```diff
@@ -107,6 +107,9 @@ def __init__(self):
                         ", ".join(unsupported_options),
                     )
 
+                if sentry_config_opts.get('before_send'):
+                    sentry_config_opts['before_send'] = import_string(sentry_config_opts['before_send'])
```

Comment:
Can you use `conf.getimport`? Thanks to this, we will have a more friendly error message.

Reply:
Good suggestion, switched it to use getimport.